axis_backend/apps/documents/views/document_viewset.py

```
@@ -0,0 +1,511 @@
+        Returns:
+            List of permission instances for current action
+        """
+        if self.action in ['update', 'partial_update', 'destroy', 'publish', 'archive', 'create_version']:
+            permission_classes = [CanManageDocuments]
+        else:
+            # list, retrieve, create - basic authenticated access
+            permission_classes = [IsAuthenticated]
+
+        return [permission() for permission in permission_classes]
+
+    def create(self, request, *args, **kwargs):
```

Ruff (0.14.5)

Missing type annotation for `*args`

(ANN002)